### PR TITLE
adding an example on virtual display

### PR DESCRIPTION
**Pull request by mw-hrastega**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -154,6 +154,31 @@ jobs:
         uses: matlab-actions/run-tests@v2
 ```
 
+## Use Virtual Display on Linux Runner
+GitHub-hosted runners based on Linux&reg; do not provide a display. To run MATLAB code that requires a display, first set up a virtual display on the runner by starting an Xvfb display server. For example, set up a virtual server to run the tests for an app created in MATLAB.
```
Comment (mcafaro) [inline profile/README.md:158]:
```suggestion
GitHub-hosted Linux runners do not provide a display. To run MATLAB code that requires a display, first set up a virtual display on the runner by starting an Xvfb display server. For example, set up a virtual server to run the tests for an app created in MATLAB.
```

**Review by mcafaro — APPROVED** [review]

File: profile/README.md (hunk 1 above)
Comment (mcafaro) [inline profile/README.md:158]: Can we add a "such as" after "To run MATLAB code that requires a display,"? Like:

To run MATLAB code that requires a display, such as UI app testing, ...

[the remark this answers is not in the document]
Comment (mw-hrastega) [reply profile/README.md]: Good catch.

Comment (mw-hrastega) [reply profile/README.md]: Good idea. Updated as:

To run MATLAB code that requires a display, such as tests that verify the correctness of an app, ..

File: profile/README.md [hunk 2]
```diff
@@ -154,6 +154,31 @@ jobs:
         uses: matlab-actions/run-tests@v2
 ```
 
+### Use Virtual Display on Linux Runner
+GitHub-hosted Linux runners do not provide a display. To run MATLAB code that requires a display, such as tests that verify the correctness of an app, first set up a virtual display on the runner by starting an Xvfb display server. For example, set up a virtual server to run the tests for an app created in MATLAB.
```
[the remark this answers is not in the document]
Comment (mw-hrastega) [reply profile/README.md:158]: Sounds good.

**Review by mcafaro — APPROVED** [review]

**Pull request merged**